Tasks/Common/azure-arm-rest/azure-arm-app-service-kudu.ts

```diff
@@ -21,14 +21,8 @@ export class KuduServiceManagementClient {
         request.headers = request.headers || {};
         request.headers["Authorization"] = "Basic " + this._accesssToken;
         request.headers['Content-Type'] = 'application/json; charset=utf-8';
-        var options: webClient.WebRequestOptions = {
```

Comment:
Why remove this?

Reply:
Here we were adding a default list of error codes which we have now added in WebClient directly.